Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Hip Tracking|Scripts|0135
+namespace VRTK
+{
+	using UnityEngine;
+	using System.Collections;
+
+
+	/// <summary>
+	/// Hip Tracking attempts to reasonably track hip position in the absence of a hip position sensor
+	/// </summary>
+	/// <remarks>
+	///
+	/// The Hip Tracking script is placed on an empty GameObject which will be positioned at the estimated hip position.
+	/// </remarks>
+
+	public class VRTK_Hip_Tracking : MonoBehaviour
+	{
+
+		private Transform playerHead;
```

Comment:
private variable should come at the end of public, then protected variables (also remove the preceeding line break)